base_dns_infrastructure/views/dns_domain_zone.xml

```diff
@@ -0,0 +1,120 @@
+                    </div>
+                    <field name="active" invisible="1" />
+                    <widget
+                        name="web_ribbon"
+                        title="Archived"
+                        bg_color="bg-danger"
+                        attrs="{'invisible': [('active', '=', True)]}"
+                    />
+                    <div class="oe_title">
+                        <label class="oe_edit_only" for="name" string="Name" />
+                        <h1>
+                            <field name="name" class="oe_inline" />
+                            <field
+                                name="parent_id"
+                                class="oe_inline"
+                                placeholder="Parent domain"
+                            />
+                        </h1>
+                    </div>
+                    <group>
+                        <group string="Commercial Informations">
+                            <field name="owner_partner_id" />
+                            <field name="renew_date" />
+                        </group>
+
+                    </group>
+
+                    <notebook>
+                        <page string="DNS Records">
+                            <field name="record_ids">
+                                <tree>
+                                    <field name="zone_id" />
+                                    <field name="type_id" />
+                                    <field name="value" />
+                                    <field name="ttl" />
+                                </tree>
+                            </field>
+                        </page>
+                    </notebook>
+                </sheet>
+                <div class="oe_chatter">
+                    <field name="message_follower_ids" />
+                    <field name="activity_ids" />
+                    <field
+                        name="message_ids"
+                        options="{'post_refresh': 'recipients'}"
+                    />
+                </div>
+            </form>
+        </field>
+    </record>
+
+    <record id="dns_zone_search" model="ir.ui.view">
+        <field name="name">DNS Zone Search</field>
+        <field name="model">dns.domain_zone</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="complete_name" />
+                <field name="owner_partner_id" />
+                <field name="renew_date" />
+                <field name="state" />
+
+                <searchpanel>
+                    <field
+                        name="parent_id"
+                        string="Domain"
+                        icon="fa-network-wired"
+                        enable_counters="1"
+                        select="one"
+                    />
+                    <field name="state" />
+                    <field
+                        name="owner_partner_id"
+                        string="Customers"
```

Comment:
on this one my idea was really to make the difference between owner and customer.
Customer is the partner which whom you make business, but you could also work as a subcontractor and want to store/share  information of the owner that could be a non-customer for you.
What's your point of view there ?

Reply:
Well. I think Owners is a correct word to use.
But I am all for consistency. I've made this remark because in the form view it is called Owner:
![image](https://github.com/OCA/infrastructure/assets/11499387/ee731c5b-4148-4c85-bce1-bac51cc12c46)

In list view it is called Customer.
![image](https://github.com/OCA/infrastructure/assets/11499387/2261163b-d70a-4563-b265-4a7924ed6af4)

Reply:
@flotho The inconsistency between the naming for the same field stille exsists.